server/database/config/connections.ts

```diff
@@ -0,0 +1,34 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const {
+  NODE_ENV, DEV_DB_URL, TEST_DB_URL, DATABASE_URL,
+} = process.env;
+
+let DB_URL = '';
+
+switch (NODE_ENV) {
+  case 'production':
+    DB_URL = DATABASE_URL;
+    break;
+  case 'development':
+    DB_URL = DEV_DB_URL;
+    break;
+  case 'test':
+    DB_URL = TEST_DB_URL;
+    break;
+  default:
+    throw new Error('No Database Found');
+}
+
+export default new Pool({
+  connectionString: DB_URL,
+  ssl:
+    process.env.NODE_ENV === 'production'
```

Comment:
You already destructured this at the top of the file

```suggestion
    NODE_ENV === 'production'
```